cycode/cli/helpers/tf_content_generator.py

```diff
@@ -0,0 +1,49 @@
+import json
+from typing import List
+
+from cycode.cli.exceptions.custom_exceptions import TfplanKeyError
+from cycode.cli.models import ResourceChange
+from cycode.cli.utils.path_utils import load_json
+
+ACTIONS_LISTS_TO_OMIT_RESOURCE = [['delete']]
+
+
+def generate_tf_content_from_tfplan(tfplan: str) -> str:
+    planned_resources = _extract_resources(tfplan)
+    return _generate_tf_content(planned_resources)
+
+
+def _generate_tf_content(resource_changes: List[ResourceChange]) -> str:
+    tf_content = ''
+    for resource_change in resource_changes:
+        if resource_change.actions not in ACTIONS_LISTS_TO_OMIT_RESOURCE:
+            tf_content += _generate_resource_content(resource_change)
+    return tf_content
+
+
+def _generate_resource_content(resource_change: ResourceChange) -> str:
+    resource_content = f'resource "{resource_change.resource_type}" "{resource_change.name}" {{\n'
+    if resource_change.values is not None:
+        for key, value in resource_change.values.items():
+            resource_content += f'  {key} = {json.dumps(value)}\n'
+    resource_content += '}\n\n'
+    return resource_content
+
+
+def _extract_resources(tfplan: str) -> List[ResourceChange]:
+    tfplan_json = load_json(tfplan)
+    resources: List[ResourceChange] = []
+    try:
+        resource_changes = tfplan_json['resource_changes']
+        for resource_change in resource_changes:
```

Comment:
possible exception `TypeError: 'NoneType' object is not iterable` is not handled. it will also occur when `_is_tfplan_document` was not called before. 

maybe `tfplan_json.get('resource_changes', [])`

or  `except (KeyError, TypeError) as e:`

Reply:
used this `except (KeyError, TypeError) as e` because we want to stop generating if `resource_changes` missing at this point